packages/ipfs/docs/MODULE.md

```diff
@@ -21,7 +21,6 @@ Use the IPFS module as a dependency of your project to spawn in process instance
   - [`options.libp2p`](#optionslibp2p)
   - [Instance methods](#instance-methods)
     - [`node.start()`](#nodestart)
-    - [`node.stop()`](#nodestop)
```

Comment:
why did you remove stop ?

Reply:
The README only has the additions to the core api - `.stop()` is documented [here](https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/MISCELLANEOUS.md#stop) so this link was broken.